src/app/docs-pages/feedback-doc/feedback-doc.component.html

```diff
@@ -4,15 +4,25 @@ <h1 class="doc-title">Feedback service
   </h1>
 
   <p class="doc-info">
-    This is the documentation for Feedback service. To use the service, call the "showMessage" method and include the arguments 
-    for message, severity, closeable and duration of the feedback message. The feedback messages appear by default in the top right corner of th UI. 
-  </p>
-  <p class="doc-info">
-    <b>
-      Typical use: When displaying some kind of information after an action. 
-      The feedback messages are categorises in four main topics; success, information, warnings and error messages.
-    </b>
+    Alert messages are messages with information that is important or time sensitive.
+    The goal is to inform and redirect. Should be clear, communicative, descriptive, short, contextual and have the right color.
   </p>
+
+  <div class="doc-info-use">
+    <h2><i class="fal fa-check-circle doc-info-use-icons icon-green"></i> <b>When to use:</b></h2>
+    <p>
+      Alert messages are used to guide the user. 
+      For an error, inform the user what the problem is. What went wrong? Why? 
+      For an alert, give them information about the status and how this can lead to an error. What can go wrong?
+    </p>
+  </div>
+  <div class="doc-info-use">
+    <h2><i class="fal fa-times-circle doc-info-use-icons icon-red"></i> <b>When not to use:</b></h2>
+    <p>
+      Alert messages should not be used unnecessarily.
```

Comment:
Mulig vi bør ha med at de bør ikke vise for mange på en gang 🤔

Reply:
Beror på use case, men ja helt enig. Beror ju på vad varningsmeddelandena visar.
Kanske viktigt att visa längre om det är så att det är superviktigt, eller om brukeren måste göra en action för att något har hänt

Reply:
"Alert messages should not be used unnecessarily. Limit the number of alerts at the same time to avoid spamming the user."

Reply:
Ja skal det bare fungere som tilbakemeldinger på actions?

Reply:
Kan for brukes til annet en resultat av actions som at du har mistet internett og er offline uten lagringsmulighet. Likte settningen din: "Alert messages should not be used unnecessarily. Limit the number of alerts at the same time to avoid spamming the user."